source/simulator/assemblers/stokes.cc

```diff
@@ -335,7 +335,7 @@ namespace aspect
                 {
                   scratch.phi_u[i_stokes] = scratch.finite_element_values[introspection.extractors.velocities].value (i,q);
                   scratch.phi_p[i_stokes] = scratch.finite_element_values[introspection.extractors.pressure].value (i, q);
-                  if (scratch.rebuild_stokes_matrix)
+                  if (scratch.rebuild_stokes_matrix || this->get_parameters().enable_elasticity)
```

Comment:
Can you please open a PR for this? I think we only need the grads though, not the div.

Reply:
add another if?
```
else if (this->get_parameters().enable_elasticity)
   {
       scratch.grads_phi_u[i_stokes] =  scratch.finite_element_values[introspection.extractors.velocities].symmetric_gradient(i,q);
    }

```